.travis.yml

```diff
@@ -5,9 +5,53 @@ cache:
   bundler: true
   directories:
     - tmp/rubocop_cache
-sudo: false
+sudo: true
 env:
+  global:
+    - KITCHEN_LOCAL_YAML=.kitchen.docker.yml
   matrix:
-    - TASK='rubocop'
-    - TASK='foodcritic'
-script: bundle exec rake $TASK
+    - TASK='rake rubocop'
+    - TASK='rake foodcritic'
+    - TASK='kitchen test nrsysmond-default-ubuntu-1204'
+    - TASK='kitchen test plugin-agent-default-ubuntu-1204'
+    - TASK='kitchen test generic-agent-default-ubuntu-1204'
+    - TASK='kitchen test php-agent-default-ubuntu-1204'
+    - TASK='kitchen test php-external-default-ubuntu-1204'
+    - TASK='kitchen test nrsysmond-default-ubuntu-1404'
+    - TASK='kitchen test plugin-agent-default-ubuntu-1404'
+    - TASK='kitchen test generic-agent-default-ubuntu-1404'
+    - TASK='kitchen test php-agent-default-ubuntu-1404'
+    - TASK='kitchen test php-external-default-ubuntu-1404'
+    - TASK='kitchen test nrsysmond-default-ubuntu-1604'
+    - TASK='kitchen test plugin-agent-default-ubuntu-1604'
+    - TASK='kitchen test generic-agent-default-ubuntu-1604'
+    - TASK='kitchen test php-agent-default-ubuntu-1604'
+    - TASK='kitchen test php-external-default-ubuntu-1604'
+    - TASK='kitchen test nrsysmond-default-debian-79'
+    - TASK='kitchen test plugin-agent-default-debian-79'
+    - TASK='kitchen test generic-agent-default-debian-79'
+    - TASK='kitchen test php-agent-default-debian-79'
+    - TASK='kitchen test php-external-default-debian-79'
+    - TASK='kitchen test nrsysmond-default-debian-84'
+    - TASK='kitchen test plugin-agent-default-debian-84'
+    - TASK='kitchen test generic-agent-default-debian-84'
+    - TASK='kitchen test php-agent-default-debian-84'
+    - TASK='kitchen test php-external-default-debian-84'
+    - TASK='kitchen test nrsysmond-default-centos-67'
+    - TASK='kitchen test plugin-agent-default-centos-67'
+    - TASK='kitchen test generic-agent-default-centos-67'
+    - TASK='kitchen test php-agent-default-centos-67'
+    - TASK='kitchen test php-external-default-centos-67'
+    - TASK='kitchen test nrsysmond-default-centos-72'
+    - TASK='kitchen test plugin-agent-default-centos-72'
+    - TASK='kitchen test generic-agent-default-centos-72'
+    - TASK='kitchen test php-agent-default-centos-72'
+    - TASK='kitchen test php-external-default-centos-72'
+
+before_script:
+- source <(curl -sL https://raw.githubusercontent.com/zuazo/kitchen-in-travis/0.5.0/scripts/start_docker.sh)
+
+script:
+- bundle exec $TASK
```

Comment:
@ofir-petrushka ahh, I see. Yeah, makes total sense to have the RuboCop & Foodcritic linting run as their own sub-builds. Worth the lack of DRY-ness.

As for “slow down package updates”, it’s not about that. The whole point of ChefDK is to provide a sensible, compatible environment as a package appropriate for developing using Chef. If you take a look at ChefCo’s own cookbooks, this is how they test, which I take as a strong recommendation: chef-cookbooks/mysql@b59be7876a31257db361c6b34e9cc6e3fbb1c041